clustintime/Processing.py

```diff
@@ -202,7 +208,7 @@ def preprocess(corr_map, analysis, data=None, window_size=1, near=1, thr=95, con
             f"Correlation with sliding window size {window_size} ",
             task,
             contrast,
-            TR
+            TR,
```

Comment:
black is opinionated on trailing commas, mainly because it lets you add/remove elements from a list without adding/removing commas.